react-hot-button-component/dist/style.css

```diff
@@ -0,0 +1,71 @@
+html {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.normal {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  border-style: hidden;
+  color: black;
+  padding: 20px 50px;
+  font-size: 30px;
```

Comment:
Making a single class that has all the common properties like `btn` then having the color classes just set the color is a better approach and makes it easier to maintain. If you wanted to change the style of the button you would have to change all the classes instead of just changing one class.